### PR TITLE
[NEW UNIVERSITY] Add new course fields to studio and add them to course search

### PR DESCRIPTION
**Pull request by arsentur**
**Description:**
- Add `topic`, `price`, `level`, `process_type` fields to courses about info
- Add them to the search, add possibility filter by new fields
- Override `edx-search` logic for processing filter facets by microsites

**Youtrack:**
[OP-105](https://youtrack.raccoongang.com/issue/OP-105)

**Configuration instructions:**
- Override edx-search=1.2.1 into requirements to rg version with this commit.
- Add needed fields to the platform by additional development
- Add to microsite needed values (ex. "COURSE_DISCOVERY_FILTERS":["topic", "level", "price", "process_type"])

**Testing instructions:**
- [x] Demo shown to QA

**Merge checklist:**
- [ ] Commit is merged - https://github.com/raccoongang/edx-search/pull/5
- [ ] New `edx-search` requirement is set in the platform requirements
- [ ] Commits are squashed
- [ ] Markup development is done (studio minor style changes)

**Post merge:**
- [ ] Delete working branch (if not needed anymore)

### CODE REVIEW

File: lms/envs/devstack.py [hunk 1]
```diff
@@ -175,17 +175,29 @@ def should_show_debug_toolbar(request):
 ########################## Course Discovery #######################
 LANGUAGE_MAP = {'terms': {lang: display for lang, display in ALL_LANGUAGES}, 'name': 'Language'}
 COURSE_DISCOVERY_MEANINGS = {
-    'org': {
```
Comment (idegtiarov) [inline lms/envs/devstack.py:178]: why?

File: lms/envs/devstack.py [hunk 2]
```diff
@@ -175,17 +175,29 @@ def should_show_debug_toolbar(request):
 ########################## Course Discovery #######################
 LANGUAGE_MAP = {'terms': {lang: display for lang, display in ALL_LANGUAGES}, 'name': 'Language'}
 COURSE_DISCOVERY_MEANINGS = {
```
Comment (idegtiarov) [inline lms/envs/devstack.py:177]: BTW why do we overwriting `COURSE_DISCOVERY_MEANINGS` If we use the same parameters as common.py set?

File: lms/envs/devstack.py (hunk 1 above)
Comment (arsentur) [reply lms/envs/devstack.py:178]: This was needed for development and testing. Removed

File: lms/envs/devstack.py (hunk 2 above)
Comment (arsentur) [reply lms/envs/devstack.py]: Removed from `devstack.py`

**Pull request merged**